Kitodo-API/src/main/java/org/kitodo/api/Metadata.java

```diff
@@ -30,9 +30,9 @@ public MdSec getDomain() {
     }
 
     /**
-     * Get the key o the metadata.
+     * Get the key o the meta-data.
```

Comment:
This change doesn't make sense - metadata is a correct word (https://whatis.techtarget.com/definition/metadata, https://www.wikiwand.com/en/Metadata)

Here something more: https://www.epublishing.com/blogs/1-the-epublishing-blog/post/143-using-metadata-for-profit
_Is it metadata or is it meta data? What is the difference between metadata versus meta data? I get this question all the time._ 

Important point meta-data is incorrect, so please don't fix something what doesn't need fix.